config/application.rb

```diff
@@ -25,12 +25,50 @@ class Application < Rails::Application
     def self.port
       if test?
         @@application_port ||= "5#{1000 + (Random.rand * 8999).to_i}".to_i
+      elsif development?
+        3000
       else
         nil
       end
     end
     
+    def self.host
+      if development? || test?
+        'localhost'
+      else
+        "http://www.helloveronica.com/"
```

Comment:
`host`  should not include protocol, that's why the `protocol` method exists below.